components/match2/wikis/apexlegends/match_summary.lua

```diff
@@ -166,6 +164,12 @@ local MATCH_STANDING_COLUMNS = {
 					return PLACEMENT_BG[opponent.placement]
 				end,
 				value = function (opponent)
+					local placementDisplay
+					if opponent.status and opponent.status ~= 'S' then
```

Comment:
instead of `'S'` should probably use the status enum here (from mgi util)

Reply:
Needs a bit bigger rework, as MGI Input Util should _not_ be imported by a match summary